src/main/java/games/strategy/engine/framework/map/download/DownloadMapsWindow.java

```diff
@@ -116,14 +117,21 @@ private DownloadMapsWindow(
     if (!pendingDownloads.isEmpty()) {
       progressPanel.download(pendingDownloads);
     }
+    // TODO: there is a possibility that pendingDownloads will contain duplicates after the following call.
+    // i don't think it matters, but we should try to avoid it if possible. might be as simply as changing
+    // to a Set.
+    pendingDownloads.addAll(DownloadCoordinator.INSTANCE.getPendingDownloads().stream()
+        .map(DownloadFile::getDownload)
+        .collect(Collectors.toList()));
+    pendingDownloads.addAll(DownloadCoordinator.INSTANCE.getActiveDownloads().stream()
+        .map(DownloadFile::getDownload)
+        .collect(Collectors.toList()));
```

Comment:
I got lazy here with these two lines of code.  They probably should be extracted to a method that explains what we're doing here.

Also, I'm not sure if `pendingDownloads` is the correct name for this variable anymore.  It's confusing to see "pending" and "active" downloads being added to a "pending" download list.  The original purpose of `pendingDownloads` was to store the maps the user requested to be downloaded when the window was opened.  Now it's storing two different types of downloads, but I don't know what the right adjective is here.  If you can think of a better name, go with it.  I'll sleep on it; worst case I'll rename it post PR.